src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/gezagvraag/GezagsBepaling.java

```diff
@@ -248,9 +246,12 @@ private boolean isOneParentPresent(Persoonslijst plOuder1, Persoonslijst plOuder
         return plOuder1 == null ^ plOuder2 == null;
     }
 
-    private HopRelatie getHopGeborenInRelatie(Persoonslijst ouder, int geboortedatum) {
-        HopRelaties hopRelaties = ouder.getHopRelaties();
-        return hopRelaties != null ? hopRelaties.geborenInRelatie(geboortedatum) : null;
+    private Optional<HopRelatie> getHopGeborenInRelatie(Persoonslijst ouder, int geboortedatum) {
+        var hopRelaties = ouder.getHopRelaties();
+        if (hopRelaties == null) return Optional.empty();
+
+        var result = hopRelaties.geborenInRelatie(geboortedatum);
+        return Optional.ofNullable(result);
```

Comment:
Suggestie uit https://github.com/BRP-API/brp-api-gezag/pull/293/files#r1930670763

```suggestion
        return Optional.ofNullable(hopRelaties.geborenInRelatie(geboortedatum));
```